Testscripts/Windows/DM-HOTADD-STRESS-NG.ps1

```diff
@@ -146,9 +146,9 @@ function Main {
         if (-not $startMemory){
            Throw "Unable to start job for creating pressure on $vm1Name" | Tee-Object -Append -file $summaryLog
         }
-        [int64]$vm1Demand = ($vmInfo.MemoryDemand/1MB)
         # Wait for stress-ng to start and the memory assigned/demand gets updated
         Start-Sleep -s $sleepTime
+        [int64]$vm1Demand = ($vmInfo.MemoryDemand/1MB)
```

Comment:
what is the difference after change this after sleep?

Reply:
1. Start-StressNg runs as background, it returns very soon. In fact, the real stress test is not running at this moment on the VM, so wait for a while is MUST. Supposed the stress needs 5s to start after Start-StressNg returns , the minimum delay is 5s.
2. The duration of Start-StressNg is 120s, the sleep time is a range from 5s~120s.  In other runtime memory test, the sleep time is 20, 50 or 80. All of them are OK.  
3. The memory decrease/increases in the VM, then the host assingns less/more memory to it. In all, $vm1Demand is dynamic change according to the memory usage in the VM.
4. According to 1, 2 and 3, you can understand why we read the MemoryDemand after some sleep. And aslo it help you understand why we run Start-StressNg in back-ground.

Reply:
Then it reads the after-value not the initial-value. Changing the order in the value access is careful.

Reply:
The test goal is that the demand memory increases when Start-StressNg is running and the demand memory decrease when Start-StressNg stops running.